services/hanke-service/src/main/resources/db/changelog/changesets/077-add-car-traffic-classification.yml

```diff
@@ -0,0 +1,38 @@
+databaseChangeLog:
+  - changeSet:
+      id: 077-add-car-traffic-classification
+      author: Teemu Hiltunen
+      changes:
+        - addColumn:
+            tableName: tormaystarkastelutulos
+            columns:
+              - column:
+                  name: haitan_kesto
+                  type: int
+                  constraints:
+                    nullable: false
+                  defaultValue: -1
```

Comment:
The default values should be removed after the columns have been added.

Reply:
Should there be any default values at all? Since we are not using my clumsy and out-of-place migration logic.

Reply:
If you delete all rows first, the defaults are not needed.

Reply:
No more default.